spec/helpers/applications_helper_spec.rb

```diff
@@ -171,6 +171,18 @@
       end
     end
 
+    describe "applications_json_path" do
```

Comment:
This should also point to the api subdomain, see https://github.com/openaustralia/planningalerts/blob/master/app/helpers/api_howto_helper.rb#L39-L42

and 

https://github.com/openaustralia/planningalerts/blob/master/app/helpers/api_howto_helper.rb#L11-L13

I think the best way to do that for now is to hard code it into a method on applications_helper like `api_host` to match the example. It should really be extracted into the theme, but if you don't feel like doing that now, I wouldn't worry about it. But implementing it the same way as in [`api_howto_helper.rb`](https://github.com/openaustralia/planningalerts/blob/master/app/helpers/api_howto_helper.rb#L11-L13) will help us extract it. If you could also add a comment like `# TODO: extract to theme` in the code, that would help.